File: src/main/java/com/mellivorines/generator/GeneratorApplication.java

package com.mellivorines.generator;

import com.github.xiaoymin.knife4j.spring.annotations.EnableKnife4j;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

/**
 * 功能描述:程序启动类
 *
 * @author lilinxi
 * @since 2022-10-19
 */
@SpringBootApplication
@EnableKnife4j
public class GeneratorApplication {

    public static void main(String[] args) {
        SpringApplication.run(GeneratorApplication.class, args);
    }

}

File: src/main/kotlin/com/mellivorines/generator/GeneratorApplication.kt
package com.mellivorines.generator

import com.github.xiaoymin.knife4j.spring.annotations.EnableKnife4j
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
@EnableKnife4j
class GeneratorApplication

fun main(args: Array<String>) {
    runApplication<GeneratorApplication>(*args)
}
